Makefile

```diff
@@ -50,8 +50,8 @@ test: start  ## Run tests
 	fi
 
 .PHONY: lint
-lint: start
-	docker-compose run --rm web flake8
+lint: 
+	docker-compose run --rm web /bin/bash -c 'flake8; mypy app --config="../mypy.ini"'
```

Comment:
Great addition! I think we should add the `--no-deps` flag, otherwise docker starts up the postgres image which we don't need here

Reply:
Ah thank you, just pushed that change.